src/System.Private.ServiceModel/tools/test/HelixLabRun/launchWCFOpenBuilds.cmd

```diff
@@ -0,0 +1,41 @@
+REM 
+REM This script sync to the latest WCF source, build and submit Helix lab runs.
+REM Arguments:
+REM %1: Lab scripts location
```

Comment:
Let's take the discussion to a different thread regarding the "git sync bootstrap" issue.  It does not need to block merging this PR.  I'm just waiting to see the changes for the other feedback before signing off.